<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PedidoRealizado</class>
 <widget class="QWidget" name="PedidoRealizado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>230</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="nombreLB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="cantidadLB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cantidad:</string>
   </property>
  </widget>
  <widget class="QLabel" name="precioLB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio Unitario: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="entregadoPB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Entregado</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eliminarPB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QLabel" name="fondoLB">
   <property name="geometry">
    <rect>
     <x>-24</x>
     <y>-8</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(120, 120, 120);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>fondoLB</zorder>
  <zorder>nombreLB</zorder>
  <zorder>cantidadLB</zorder>
  <zorder>precioLB</zorder>
  <zorder>entregadoPB</zorder>
  <zorder>eliminarPB</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
